server/util.js

```diff
@@ -0,0 +1,14 @@
+const isArrayOfStrings = (arr) => {
+    if (!Array.isArray(arr)) {
+        return false;
+    }
+    let valid = true;
+    arr.map((ele) => {
+        if (typeof ele !== "string") {
+            valid = false;
+        }
+    });
+    return valid;
```

Comment:
This would be a good use case for the array.every() function

Reply:
Thanks for the tip. I rewrote it to be much smaller & easier to read.
```js
const isArrayOfStrings = (arr) =>
    Array.isArray(arr) && arr.every((ele) => typeof ele === "string");
```